### PR TITLE
Awesome books: Refactor to use javascript classes

### PR DESCRIPTION
**Pull request by prolajumokeoni**
In this project:
- refactored all objects to use javascript classes and class methods

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,66 +1,80 @@
...
+  static removeBook(author) {
+    const newBooks = Storage.getBooks();
+    newBooks.forEach((book, index) => {
+      if (book.author === author) {
+        newBooks.splice(index, 1);
+      }
+    });
+    localStorage.setItem('books', JSON.stringify(newBooks));
+  }
+}
```
Comment (vichuge) [inline index.js]: - [ ] There's a bug when you remove book, to see it follow this steps:

1. Create 5 books with the same author and title, I use `title: aaaaa author: aaa`
2. Delete the first one you created, the book on the top
3. Refresh the page

If you do it you could see how another book is deleted. Please fix this.
- Hint: A good way to fix it is to assign ids for each element on the list, and then delete it according to the id in place author or title. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your README and PR message are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (Kalunge) [reply index.js]: Thanks for the review, But why would anyone need to add the same book 5 times in a real-life situation. Those fields would be marked as unique

**Review by henatan99 — COMMENTED** [review]
Hi @prolajumokeoni 

# Status: Invalid :x: 

- [ ] I am sorry I can not proceed to review your project because you have failing eslint checks. Kindly fix that and submit it for review. This submission will not be counted towards the CR limit.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._Duplicate of #

File: index.js (hunk 1 above)
Comment (sinansevgi) [reply index.js]: [Optional]
Hello @Kalunge,  Your fix is enough to fix the bug. But you should also consider books with the same title (or I can have multiple copies of the same book). Let's say I have _The Double_ from _Fyodor Dostoevsky_ and _The Double_ from _José Saramago_, if you set the field to unique there is no way for me to add both of these books. 

As @vichuge suggested the best way to solve this issue is using unique `ids` for each record.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Kalunge and @prolajumokeoni , 

Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**